cdc-accelerator/feature/test.gigyaResponses.js

```diff
@@ -225,9 +225,6 @@ export const expectedPoliciesResponse = {
         verificationEmailExpiration: 86400,
         autoLogin: false,
         defaultLanguage: 'en',
-        emailTemplates: {
```

Comment:
This file contains the gygia responses mocks. 
If we remove the email templates from the policies response then there's nothing to do in the code, the emails won't be written on the policies.json.
Do not remove the templates from the policies response